tests/command.deploy.test.js

```diff
@@ -92,6 +92,7 @@ if (process.env.NETLIFY_TEST_DISABLE_LIVE !== 'true') {
     })
   })
 
+  // TODO: check with erez if we should update edge handler min version to 12 so they can drop compatibility
```

Comment:
For the CLI tests we should drop this condition (we should have dropped it a while back).

I suggest opening a new issue in https://github.com/netlify/netlify-plugin-edge-handlers to handle Node.js 10 deprecation